test/README.md

```diff
@@ -0,0 +1,97 @@
+# Running Tests
+
+To run all tests, perform the following steps:
+
+## Startup a fresh iRODS server
+
+Use the iRODS testing environment to setup an iRODS server: https://github.com/irods/irods_testing_environment
+
+### Get the iRODS network information
+
+We need the network that the HTTP API is running on to have all pieces of the server
+communicate with each other.
+
+Following is an example of how to list all available docker networks with examples
+of how the output may appear:
+
+```console
+$ docker network ls
+NETWORK ID     NAME                              DRIVER    SCOPE
+d06849108e9c   bridge                            bridge    local
+371117285a13   host                              host      local
+fbf586b6459f   none                              null      local
+297992bf57a9   ubuntu-2204-postgres-14_default   bridge    local
+```
+
+In the example output, the desired network would be `ubuntu-2204-postgres-14_default`.
+
+The docker network used will be referred to as `<NETWORK>` in the following sections.
+
+## Startup the Keycloak image for testing
+
+See [Build the Testing Image](#build-the-testing-image) to build the image first.
```

Comment:
Why not move the build section here instead of having the reader jump to the bottom of the README?